include/service/entities/custom_object_service.js

```diff
@@ -955,7 +955,7 @@ module.exports = function CustomObjectServiceModule(pb) {
             options = {};
         }
 
-        if (!pb.validation.isIdStr(id, true)) {
+        if (!pb.validation.isIdStr(id.toString(), true)) {
```

Comment:
You should be able to change this to ```if (!pb.validation.isId(id, true)) {```.  Otherwise you risk a null reference error